packages/@aws-cdk/aws-rds/lib/instance-engine.ts

```diff
@@ -103,22 +198,134 @@ class MariaDbInstanceEngine extends InstanceEngineBase {
       engineType: 'mariadb',
       singleUserRotationApplication: secretsmanager.SecretRotationApplication.MARIADB_ROTATION_SINGLE_USER,
       multiUserRotationApplication: secretsmanager.SecretRotationApplication.MARIADB_ROTATION_MULTI_USER,
-      parameterGroupFamilies: [
-        { engineMajorVersion: '10.0', parameterGroupFamily: 'mariadb10.0' },
-        { engineMajorVersion: '10.1', parameterGroupFamily: 'mariadb10.1' },
-        { engineMajorVersion: '10.2', parameterGroupFamily: 'mariadb10.2' },
-        { engineMajorVersion: '10.3', parameterGroupFamily: 'mariadb10.3' },
-      ],
-      version: props.version,
+      version: props.version
+        ? {
+          fullVersion: props.version.mariaDbFullVersion,
+          majorVersion: props.version.mariaDbMajorVersion,
+        }
+        : {
+          majorVersion: '10.3',
+        },
     });
   }
 }
 
+/**
+ * The versions for the MySQL instance engines
+ * (those returned by {@link DatabaseInstanceEngine.mysql}).
+ */
+export class MysqlEngineVersion {
+  /** Version "5.5" (only a major version, without a specific minor version). */
+  public static readonly VER_5_5 = MysqlEngineVersion.of('5.5', '5.5');
```

Comment:
Specifying only the major version works but should there be a static property for that? It's not recommended and the console experience won't let you that.

Reply:
Hmm, that's interesting. I didn't find anything in the documentation that suggests that it's not safe. Do you have some reference for that @jogold ?

Reply:
It's not that it's not safe, it's just that you don't know what you'll get as final version. And if you deploy the same stack later (additional stage, region) you could get another version for the same code.

But leaving it is a good compromise if you now make `version` required in all static engine construction methods.